src/script/page/LeftSidebar/panels/Preferences.tsx

```diff
@@ -79,6 +129,14 @@ const Preferences: React.FC<PreferencesProps> = ({contentViewModel, teamReposito
     contentViewModel.switchContent(item.id);
   };
 
+  useEffect(() => {
+    if (NEW_DEVICE_NOTIFICATION_STATES.includes(contentState)) {
```

Comment:
This is the actual fix (also triggering the modal when content state is `account_preferences` or `device_preferences`)